core/state-machine/src/testing.rs

```diff
@@ -92,9 +88,29 @@ impl<H: Hasher, N: ChangesTrieBlockNumber> TestExternalities<H, N> {
 	pub fn changes_trie_storage(&mut self) -> &mut ChangesTrieInMemoryStorage<H, N> {
 		&mut self.changes_trie_storage
 	}
+
+	/// Return a new backend with all pending value.
+	pub fn commit_all(&self) -> InMemory<H> {
+		self.backend.update(
+			core::iter::empty()
+				.chain(self.overlay.committed.top.clone().into_iter()
+					.map(|(k, v)| (None, k, v.value)))
```

Comment:
```suggestion
					.map(|(k, v)| (None, k, v.value))
				)
```

The rest should be formatted the same way.